src/components/planning/DayPartSettings.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import styled from "styled-components"
+import {Select} from "@datapunt/asc-ui"
+import {FormField} from "../form-components/FormComponents"
+import {StadiumSettings} from "./StadiumSettings"
+
+type Props = {
+  index: number
+  day: string
+}
+
+const Wrap = styled.div`
+  padding: 20px 0;
+`
+
+const H4 = styled.h4`
+  margin: 18px 0 4px
+`
+
+const OPTIONS = [
+  'Onderzoek buitendienst',
+  '2de Controle',
+  '3de Controle',
+  'Hercontrole',
+  '2de hercontrole',
+  '3de hercontrole',
+  'Avondronde',
+  'Onderzoek advertentie',
+  'Weekend buitendienstonderzoek',
+  'Issuemelding'
+]
```

Comment:
These options are being loaded from the backend. Should not be hardcoded.

Reply:
Done